lang/csharp/README.md

```diff
@@ -63,7 +63,12 @@ Example:
   </ItemGroup>
 
   <Target Name="AvroGenerate" BeforeTargets="CoreCompile">
-    <AvroGenTask SchemaFiles="@(SchemaFiles)" OutDir="generated" />
+    <AvroGenTask SchemaFiles="@(SchemaFiles)" OutDir="generated" >
+      <!-- bind generated files task output -->
+      <Output ItemName="GeneratedCodeFiles" TaskParameter="GeneratedFiles" />
+    </AvroGenTask>
+    <!-- displays generated files in build logs -->
+    <Message Text="Generated: %(GeneratedCodeFiles.Identity)" Importance="normal" />
```

Comment:
The generated files are always a realtive path to the OutDir, and the path starting with OutDir. Havent tried it, but `WriteTypes` just combines OutDir with the extra path (namespaces). There is no conversion to absolute path, so that relative path will be  returned in the `GeneratedFiles` list.